rib/rib.go

```diff
@@ -79,7 +79,7 @@ type RIBHookFn func(constants.OpType, int64, string, ygot.ValidatedGoStruct)
 //   - a string indicating the network instance that the operation was within
 //   - an enumerated value indicating the AFT the operation was within.
 //   - an any that indicates the impacted AFT entry's key. The function must cast
-//     the any to the relevant type.
```

Comment:
I think `the any` or `the key` is valid here -- since we're talking about the `any` argument.